modules/light-clients/08-wasm/types/errors.go

```diff
@@ -15,4 +15,5 @@ var (
 	ErrWasmSubMessagesNotAllowed = errorsmod.Register(ModuleName, 10, "execution of sub messages is not allowed")
 	ErrWasmEventsNotAllowed      = errorsmod.Register(ModuleName, 11, "returning events from a contract is not allowed")
 	ErrWasmAttributesNotAllowed  = errorsmod.Register(ModuleName, 12, "returning attributes from a contract is not allowed")
+	ErrRetrieveClientID          = errorsmod.Register(ModuleName, 13, "failed to retrieve client id")
```

Comment:
I just pushed a small change to move this error up, since it's not wasm specific.

Reply:
Technically, it is emitted even when the clientID does not have `08-wasm`, so it is kinda wasm specific but I see your point. I'll leave your push as is